### PR TITLE
Update aggregations.html

### PR DESCRIPTION
**Pull request by lmuhlha**

### CODE REVIEW

File: docs/content/_docs/aggregations.html [hunk 1]
```diff
@@ -54,30 +58,108 @@ <h3>Size and Extent</h3>
...
+<h3 id="average">
+  Average Aggregation
+  <a class="link-to" href="#average"><span class="glyphicon glyphicon-link"></span></a>
 </h3>
 
 <h5>JSON</h5>
 
 <pre><code class="language-json">
-{"type": "min", "sampling": {"unit": &lt;unit&gt;, "size": &lt;number&gt;, "extent": &lt;number&gt;}}
+{"type": "average", "sampling": {"unit": &lt;unit&gt;, "size": &lt;number&gt;, "extent": &lt;number&gt;}}
```
Comment (hexedpackets) [inline docs/content/_docs/aggregations.html]: We should update the sampling definition for the aggregations, `size` was changed to `value` a while ago and `extent` isn't always needed. In grafana we set `unit` and `value`, for example `{"type": "average", "sampling": {"unit": "minutes", "value": 2}}`

File: docs/content/_docs/aggregations.html [hunk 2]
```diff
@@ -102,125 +184,131 @@ <h5>Description</h5>
...
+<h3 id="notneg">
+  Not Negative Aggregation
+  <a class="link-to" href="#notneg"><span class="glyphicon glyphicon-link"></span></a>
 </h3>
 
 <h5>JSON</h5>
 
 <pre><code class="language-json">
-{"type": "sum", "sampling": {"unit": &lt;unit&gt;, "size": &lt;number&gt;, "extent": &lt;number&gt;}}
+{"type": "notNegative", ()}
```
Comment (hexedpackets) [inline docs/content/_docs/aggregations.html]: Are the parens valid JSON?

File: docs/content/_docs/aggregations.html [hunk 3]
```diff
@@ -54,30 +58,108 @@ <h3>Size and Extent</h3>
 
 <img style="width: 100%;" src="{{ 'images/size_extent_3.svg' | relative_url }}"></img>
 
-<h3 id="min">
-  Min Aggregation
-  <a class="link-to" href="#min"><span class="glyphicon glyphicon-link"></span></a>
+## AVERAGE
```
[the remark this answers is not in the document]
Comment (lmuhlha) [reply docs/content/_docs/aggregations.html]: meant to make em comments, can just remove them. was for my own readability.

**Comment by lmuhlha** [comment]
@hexedpackets made the fixes

**Pull request merged**